tst/regression/test_suites/amr_performance/amr_performance.py

```diff
@@ -0,0 +1,120 @@
+#========================================================================================
+# Parthenon performance portable AMR framework
+# Copyright(C) 2021 The Parthenon collaboration
+# Licensed under the 3-clause BSD License, see LICENSE file for details
+#========================================================================================
+# (C) (or copyright) 2021. Triad National Security, LLC. All rights reserved.
+#
+# This program was produced under U.S. Government contract 89233218CNA000001 for Los
+# Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+# for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+# in the program are reserved by Triad National Security, LLC, and the U.S. Department
+# of Energy/National Nuclear Security Administration. The Government is granted for
+# itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+# license in this material to reproduce, prepare derivative works, distribute copies to
+# the public, perform publicly and display publicly, and to permit others to do so.
+#========================================================================================
+
+# Modules
+import math
+import numpy as np
+import matplotlib
+matplotlib.use('agg')
+import matplotlib.pylab as plt
+import sys
+import os
+import utils.test_case
+
+""" To prevent littering up imported folders with .pyc files or __pycache_ folder"""
+sys.dont_write_bytecode = True
+
+
+class TestCase(utils.test_case.TestCaseAbs):
+    def Prepare(self, parameters, step):
+        # making sure env variable is set to produce json output from kokkos space-time-stack
+        os.environ['KOKKOS_PROFILE_EXPORT_JSON'] = 'ON'
+        
+        msg = "Missing Kokkos profiling lib.\n"
+        msg += "Please set KOKKOS_PROFILE_LIBRARY to kp_space_time_stack.so"
+        assert parameters.kokkos_profiling_lib is not None, msg
+
+        # Profiling libraries introduce fences etc.
+        # Thus, we run once with and once without.
+        if step % 2 != 0:
+            os.unsetenv('KOKKOS_PROFILE_LIBRARY')
+
+        if step == 1 or step == 2:
```

Comment:
I think the following might be clearer if it works:
```suggestion
        if 1 <= step <= 2:
```
and likewise below